resources/assets/components/AffiliateOptInToggle/AffiliateOptInToggle.js

```diff
@@ -21,7 +22,7 @@ const AffiliateOptInToggle = ({
         className="form-checkbox opt_in_checkbox"
         onClick={clickedOptIn}
       />
-      <TextContent styles={{ textColor: '#fff' }}>
+      <TextContent styles={{ textColor: `${textColor}` }}>
```

Comment:
This doesn't need to be a template string!

```suggestion
      <TextContent styles={{ textColor }}>
```